Extensions/shuffle+.js

```diff
@@ -116,6 +116,10 @@
             case Spicetify.URI.Type.TRACK:
             case Spicetify.URI.Type.EPISODE:
                 return [uri];
+            case Spicetify.URI.Type.STATION:
+            case Spicetify.URI.Type.RADIO:
+                playerPlayOGFunc({ uri: uri }, { featureVersion: Spicetify.Platform.PlayerAPI._defaultFeatureVersion });
+                return ["playedstation"];
```

Comment:
It is a workaround, but I think it is very valid.
Do we need this line though? 
```rust
playerPlayOGFunc({ uri: uri }, { featureVersion: Spicetify.Platform.PlayerAPI._defaultFeatureVersion });
```

Reply:
Yeah, this is the only way to play stations, I tested it out a lot of different ones, but this is the only one that works.

**For a bit of context**: 
As I am sure you're aware, shuffle+ injects its own playing function when auto-shuffle is on, thankfully it does store the original function, which is, as the name suggests `playerPlayOGFunc`, so I have to use this to play the stations. And those arguments is the only way I know how to call it successfully.

Reply:
I was actually not aware when I made the comment as I haven't messed around with shuffle+, but afterwards I went digging into the code and found that binding. Thanks for the context!!